### PR TITLE
integrate screen sharing tracker window

### PR DESCRIPTION
**Pull request by andrei-gavrilescu**
This PR depends on https://github.com/jitsi/jitsi-meet-electron-utils/pull/66.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6052,21 +5976,13 @@
       }
     },
     "jitsi-meet-electron-utils": {
-      "version": "github:jitsi/jitsi-meet-electron-utils#3b6562f4d6104bfb5099414d1a79d4164767ea9b",
```
Comment (hristoterezov) [inline package-lock.json]: I've just created a jitsi-meet-electron-utils release. Could you please rerun npm install?

File: main.js [hunk 2]
```diff
@@ -149,6 +150,7 @@ function createJitsiMeetWindow() {
     initPopupsConfigurationMain(mainWindow);
     setupAlwaysOnTopMain(mainWindow);
     setupPowerMonitorMain(mainWindow);
+    setupScreenSharingMain(mainWindow, 'Jitsi Meet');
```
Comment (saghul) [inline main.js]: Please don't hardcode this. Use https://github.com/jitsi/jitsi-meet-electron/blob/3c448837e771796018dfff46dda2b89c7e5f98e2/app/features/config/index.js#L11

Comment (andrei-gavrilescu) [reply main.js]: ok, will create a PR to fix this.

Comment (saghul) [reply main.js]: Thanks!

**Pull request merged**